src/components/pages/Navbar/NavbarFeatures/LogoutButton.js

```diff
@@ -1,24 +1,20 @@
 import React from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
-
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function LogoutButton() {
   const { logout, isAuthenticated } = useAuth0();
 
   const logoutAuth = () => {
     localStorage.removeItem('AuthToken');
-    logout();
+    logout({ returnTo: window.location.origin });
```

Comment:
I enjoy the efficieny in this function. By making the logoutAuth clear the AuthToken and redirecting the user to the home page, you knocked out several lines of code.